src/azure-cli/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -286,7 +297,12 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('node_type_name', options_list=['-n', '--name', '--node-type-name'], help='node type name.')
         c.argument('instance_count', help='essage = "The number of nodes in the node type.')
         c.argument('primary', arg_type=get_three_state_flag(), help='Specify if the node type is primary. On this node type will run system services. Only one node type should be marked as primary. Primary node type cannot be deleted or changed for existing clusters.')
-        c.argument('disk_size', help='Disk size for each vm in the node type in GBs.', default=100)
+        c.argument('disk_size', options_list=['--disk-size', '--data-disk-size'], help='Disk size for each vm in the node type in GBs.', default=100)
```

Comment:
should this parameter be int type?

Reply:
yes. how should I define it as int? I don't see any other int param having anything special

Reply:
c.argument('disk_size', type=int,

Reply:
added thanks